docs/api/Client.md

```diff
@@ -28,6 +28,8 @@ Returns: `Client`
 * **connect** `ConnectOptions | Function | null` (optional) - Default: `null`.
 * **strictContentLength** `Boolean` (optional) - Default: `true` - Whether to treat request content length mismatches as errors. If true, an error is thrown when the request content-length header doesn't match the length of the request body.
 * **interceptors** `{ Client: DispatchInterceptor[] }` - Default: `[RedirectInterceptor]` - A list of interceptors that are applied to the dispatch method. Additional logic can be applied (such as, but not limited to: 302 status code handling, authentication, cookies, compression and caching). Note that the behavior of interceptors is Experimental and might change at any given time.
+* **autoSelectFamily**: `boolean` (optional) - Enables a family autodetection algorithm that loosely implements section 5 of [RFC 8305](https://tools.ietf.org/html/rfc8305#section-5). See [here](https://nodejs.org/api/net.html#socketconnectoptions-connectlistener) for more details. This option is ignored if not supported by the current Node version.
+* **autoSelectFamilyAttemptTimeout**: `number` - The amount of time in milliseconds to wait for a connection attempt to finish before trying the next address when using the `autoSelectFamily` option. See [here](https://nodejs.org/api/net.html#socketconnectoptions-connectlistener) for more details.
```

Comment:
What's the default?

Reply:
For `autoSelectFamily` is none (which is equivalent for to false).
For `autoSelectFamilyAttemptTimeout` is 250, but in Node, not in undici. I simply forward the option to node and that's why I haven't mentioned it here.

Reply:
Should we maybe enable it as default? @mcollina

Reply:
@ShogunPanda then at least mention it uses the defaults from node otherwise?

Reply:
Absolutely. I just added it.